packages/react-instantsearch/src/connectors/connectRefinementList.js

```diff
@@ -100,9 +100,23 @@ export default createConnector({
         isRefined: v.isRefined,
       }));
 
+    let facetValues;
+    if (facetValuesResults) {
+      facetValues = facetValuesResults[attributeName]
+        .slice(0, limit).map(
+          v => ({
+            label: v.value,
+            value: getValue(v.value, props, searchState),
+            _highlightResult: {label: {value: v.highlighted}},
```

Comment:
To use the Highlight widget when displaying the search for values results, I need to cheat a little to mimic the regular hits structure. Do you think it's fine to do that here, or should we:
- have the same structure between hits and facetValues in the helper regarding the highlighting
- have several implementations of the higlight in react-instantsearch